src/components/change-requests/change-requests.tsx

```diff
@@ -8,11 +8,18 @@ import { routes } from '../../shared/routes';
 import ChangeRequestsTable from './change-requests-table/change-requests-table';
 import ChangeRequestDetails from './change-request-details/change-request-details';
 import NewChangeRequest from './new-change-request/new-change-request';
+import ReviewChangeRequest from './review-change-request/review-change-request';
 import './change-requests.module.css';
 
 const ChangeRequests: React.FC = () => {
   return (
     <Switch>
+      <Route path={routes.CHANGE_REQUESTS_ACCEPT}>
+        <ReviewChangeRequest accepted={true} />
+      </Route>
+      <Route path={routes.CHANGE_REQUESTS_DENY}>
+        <ReviewChangeRequest accepted={false} />
```

Comment:
```suggestion
        <ReviewChangeRequest accept />
      </Route>
      <Route path={routes.CHANGE_REQUESTS_DENY}>
        <ReviewChangeRequest accept={false} />
```

I would also suggest renaming the prop so it's a little bit clearer since the component provides the user with the _option_ to accept the CR

Reply:
Do you think renaming the prop to `option` and having values of `'accept'` or `'deny'` be better and more readable?

Reply:
Love it!